lib/internal/process.js

```diff
@@ -181,37 +181,64 @@ function setupSignalHandlers() {
   // Load events module in order to access prototype elements on process like
   // process.addListener.
   const signalWraps = {};
+  const oneshots = {};
```

Comment:
Hm, I know the signalWraps use a object here but would it not be nicer to use a `Set` here?

Reply:
Maybe I'm missing something but wouldn't it need to be a `Map`?

Reply:
Yes, my bad.